### DISCUSSION TITLE
Revisiting Derivatives Exercise Question

### DISCUSSION THREAD

**Discussion by Aarons-Joshua** [cooperrc/computational-mechanics#64] [Q&A]
Hey guys. Did anyone figure out how to calculate the error between the actual and analytical fx's?

What is the maximum error between the numerical  𝑑𝑓𝑑𝑥  and the actual  𝑑𝑓𝑑𝑥 ?

**Comment by cooperrc** [comment]
Can you share the work you have so far? I'm not sure what question you're working on.

**Reply by Aarons-Joshua** [reply]
Of course, it is in the lecture notes part 1. 
![image](https://user-images.githubusercontent.com/98834673/176224955-6cc34617-a86a-4621-84ee-e326aa33ff50.png)
